packages/core/interfaces/src/index.ts

```diff
@@ -15,6 +15,7 @@ export interface UseProduct<PRODUCT> {
   }) => Promise<void>;
   loading: ComputedProperty<boolean>;
   [x: string]: any;
```

Comment:
are we going to not using the error ref in case of problems? (regarding all composables which can do the DAO calls)